package.json

```diff
@@ -18,7 +18,9 @@
     "buffer-xor": "^2.0.2",
     "envelope-spec": "^1.0.0",
     "futoin-hkdf": "^1.3.2",
-    "sodium-native": "^3.2.0"
+    "sodium-native": "^3.2.0",
+    "ssb-bfe": "^0.5.0",
```

Comment:
looks like this is a devDep at the moment?
Is that right?

Reply:
Right, thanks!